tutorials/Consistent_Scores.ipynb

```diff
@@ -0,0 +1,452 @@
+    "\n",
+    "\n",
+    "You may already know that the MSE is a consistent score for forecasting the mean and the MAE is a consistent score for forecasting the median. But did you know that there are a whole family of scores that are consistent for prediciting the mean, median, or quantiles? \n",
+    "\n",
+    "The consistent scoring module in `scores` provides access to these families of consistent scores that can be used to emphasise predictive performance across desired decision thresholds.\n",
+    "\n",
+    "### Example 1. A score consistent with the mean\n",
+    "Let's jump into an example that illustrates a scoring function that is consistent with predicting the mean value using `scores`. Suppose we want to evaluate the performance of expected daily precipitation forecasts, but we want to place increasing importance on correctly forecasting extreme values. \n",
+    "\n",
+    "\n",
+    "We want to used the `consistent_expectile_score` in `scores` to evaluate the expected daily precipitation forecasts. Note that medians relate to quantiles in an analogous way to how the mean relates to [expectiles](https://en.wikipedia.org/wiki/Expectile).\n",
+    "\n",
+    "Our consistent scoring function for expectiles is \n",
+    "\n",
+    " $$ S(x, y) =\n",
+    "    \\begin{cases}\n",
+    "    (1 - \\alpha)(\\phi(y) - \\phi(x) - \\phi'(x)(y-x)), & y < x \\\\\n",
+    "    \\alpha(\\phi(y) - \\phi(x) - \\phi'(x)(y-x)), & x \\leq y\n",
+    "    \\end{cases}\n",
+    "$$\n",
+    "\n",
+    "Where:\n",
+    "- $x$ is the forecast\n",
+    "- $y$ is the observation\n",
+    "- $\\alpha$ is the expectile level\n",
+    "- $\\phi$ is a [convex function](https://en.wikipedia.org/wiki/Convex_function) of a single variable\n",
+    "- $\\phi'$ is the [subderivative](https://en.wikipedia.org/wiki/Subderivative) of $\\phi$. The subderivative is a generalisation of the derivative for convex functions and coincides with the derivative when the convex function is differentiable\n",
+    "- $S(x,y)$ is the score.\n",
+    "\n",
+    "To make the expectile the consistent with the mean, set $\\alpha=0.5$.\n",
+    "\n",
+    "Next we need to determine our $\\phi(z)$ and $\\phi'(z)$ functions. Let's assume that that the importance of predicting extreme rainfall increases exponentially.\n",
+    "\n",
+    "First we create the weighting function\n",
+    "\n",
+    "$$\\phi''(z) = e^\\frac{z}{10}$$ \n",
+    "\n",
+    "This will place increasing importance on more extreme rainfall thresholds. We will plot this weighting function further below.\n",
+    "\n",
+    "Next we need to integrate $\\phi''(z)$ twice so that we can obtain the functions for our consistent scoring function\n",
+    "\n",
+    "$$\\phi'(z) = 10e^\\frac{z}{10}$$ \n",
+    "and \n",
+    "$$\\phi(z) = z + 100e^\\frac{z}{10}$$ \n",
```

Comment:
Was there a reason for the $z$ term here? It can be omitted, because if you substitute it into the formula for S you get cancellation (as though you never had a z term).